draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1572,13 +1572,20 @@ elements.
 
 ### Receiving SCTs
 
-TLS clients receive SCTs alongside or in certificates. TLS clients MUST
-implement all of the three mechanisms by which TLS servers may present SCTs (see
-{{tls_servers}}). TLS clients MAY also accept SCTs via the `status_request_v2`
-extension ([RFC6961]). TLS clients that support the `transparency_info` TLS
-extension SHOULD include it in ClientHello messages, with empty
-`extension_data`. TLS clients may also receive inclusion proofs in addition to
-SCTs, which should be checked once the SCTs are validated.
+Servers may provide evidence of logging via any of the channels described in
+{{tls_servers}}.  In order to ensure that any information sent by the server is
+successfully received by the client, clients that intend to evaluate compliance
+MUST support both the `transparency_info` and `status_request` TLS extensions.
```

Comment:
Why add this paragraph?  The first sentence is irrelevant to this section.  The second sentence is already covered by the next paragraph.